lnurl/pay.js

```diff
@@ -62,16 +57,6 @@ module.exports = (args, cbk) => {
           return cbk([400, 'ExpectedUrlToGetPaymentRequestFromLnurl']);
         }
 
-        try {
```

Comment:
should still validate the url here, otherwise I think the error won't be cleanly represented